README.md

```diff
@@ -1,11 +1,72 @@
 [![Build Status](https://travis-ci.com/SUSE-Enceladus/public-cloud-info-client.svg?branch=master)](https://travis-ci.com/SUSE-Enceladus/public-cloud-info-client)
 
-pint
-====
+# Introduction
 
 pint (Public Cloud INformation Tracker) is a command line client to access
 the REST API provided by the SUSE Public Cloud Information Service. The
 information service provides data about the images SUSE publishes in various
 public cloud frameworks (Amazon, Google, Microsoft) as well as the
 servers that make up the SUSE maintained and operated update infrastructure
 in those frameworks.
+
+## Quick Start
+
+There are various ways to install the Pint Client:
+
+### Install the python package using pip
+
+```bash   
+  $ pip install susepubliccloudinfo
+```
+
+### Install On SUSE Linux using rpm package  
+  
+```bash
+  $ zypper in python3-susepubliccloudinfo
+```
+
+### Install from Source
+
+```bash
+  $ git clone https://github.com/SUSE-Enceladus/public-cloud-info-client
+  $ cd public-cloud-info-client
+
+  $ pip install -e .[dev]
+```
+
+## Usage
+
+```bash
+  usage: pint -h | --help
+       pint providers
+          [ --json | --xml ]
+       pint image_states
+          [ --json | --xml ]
+       pint (google|alibaba|amazon|oracle|microsoft) server_types 
+          [ --json | --xml ]
+       pint (google|alibaba|amazon|oracle|microsoft) regions
+          [ --filter=<filter> ]
+          [ --json | --xml ]
+       pint (google|alibaba|amazon|oracle|microsoft) servers
+          [ --filter=<filter> ]
+          [ --json | --xml ]
+          [ --region=<region> ]
+          [ --smt | --regionserver ]
+       pint (google|alibaba|amazon|oracle|microsoft) images
+          [ --active | --inactive | --deleted | --deprecated ]
+          [ --filter=<filter> ]
+          [ --json | --xml ]
+          [ --region=<region> ]
+       pint -v | --version
+```  
+
```

Comment:
Echoing the help into the README means we have to touch the README every time there is a change to the cli. This will be forgotten and thus get things out of sync. We should not deliberately create more sources for bugs or potential misinformation.

Reply:
Thats a good point and my fault for one of the acceptance criteria in the ticket...  Perhaps just reference the help command here